core/src/main/resources/hudson/model/Job/configure.properties

```diff
@@ -21,5 +21,5 @@
 # THE SOFTWARE.
 
 name={0} name
-Save=Save All
-Apply=Apply All
\ No newline at end of file
+Save=Save
+Apply=Apply
```

Comment:
@jglick  We had, but we (group including KK and Daniel, I think) decided to change it back. The reason for it was that classic tabs have an ambiguity about what is being saved, 1 tab panel or all tab panels. The scrolling tabs don't share that issue.